rules/cloud/azure/azure_ad_auth_failure_increase.yml

```diff
@@ -0,0 +1,22 @@
+title: Increased Failed Authentications Of Any Type
+id: e1d02b53-c03c-4948-b11d-4d00cca49d03
+status: experimental
+description: Detects when sign-ins increased by 10% or greater.
+author: Mark Morowczynski '@markmorow', MikeDuddington, '@dudders1'
+date: 2022/08/11
+references:
+    - https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/security-operations-user-accounts#monitoring-for-failed-unusual-sign-ins
+logsource:
+    product: azure
+    service: signinlogs
+detection:
+    selection:
+        Status: failure
+        Count: "<10%"
+    condition: selection
+falsepositives:
+     - none
```

Comment:
"None" isn't a valid value for FP. Unlikely is the preferred option

Reply:
ok updated